### PR TITLE
Rewrite with `jsPDF`

### PR DESCRIPTION
**Pull request by zoey-kaiser**
Rewrites the core logic with jspdf rather then html2pdf.

### CODE REVIEW

File: src/runtime/composables/usePDFExport.ts [hunk 1]
```diff
@@ -1,5 +1,20 @@
-import { useNuxtApp } from '#app'
-import { Html2PdfOptions } from '../plugin.client'
-export const usePDFExport = async (element: HTMLElement, options: Html2PdfOptions) => {
-  await useNuxtApp().$exportToPDF(element, options)
+import jsPDF, { HTMLOptions, jsPDFOptions } from 'jspdf'
+
+export type PDFOptions = HTMLOptions
```
Comment (valiafetisov) [inline src/runtime/composables/usePDFExport.ts:3]:
```suggestion
```

**Review by valiafetisov — COMMENTED** [review]
Congrats with a great (and working) package 🎉 🎉 🎉 

I think the exciting next step is to figure out [server-side rendering](https://github.com/parallax/jsPDF/issues/2248) using the same package ✨

**Comment by zoey-kaiser** [comment]
> I think the exciting next step is to figure out [server-side rendering](https://github.com/parallax/jsPDF/issues/2248) using the same package ✨

Definitely agree! I will work on this once we release a stable version!

**Comment by zoey-kaiser** [comment]
TODO: Change version number when we are ready to release

**Review by BracketJohn — APPROVED** [review]
LGTM - are there any improvements that were made? E.g., could we deprecate some of the style hacks / switch to composable only / ...?

If so: let's document them

**Comment by zoey-kaiser** [comment]
> LGTM - are there any improvements that were made? E.g., could we deprecate some of the style hacks / switch to composable only / ...?

I have already removed them from the docs and the playground. I may add examples again later, however I wanted to get a stable release out beforehand 🤗

**Review by BracketJohn — APPROVED** [review]

**Pull request merged**